Config/Glide64.rdb

```diff
@@ -2965,6 +2965,15 @@ Good Name=Shadow Man (U)
 Internal Name=Shadowman
 depthmode=0
 
+// added 2015.04.11:  "Tsumi to Batsu:  Hoshi no Keishousha" English translation
+// [B6BC0FB0-E3812198-C:4A]
```

Comment:
No.

I put them there just to help prevent future commits from changing the settings for only one version and not the other, since this would be one of the first--if not the only--new entry in the Glide64 RDB that wasn't in the official Glide64 INI.  I was just trying to prevent confusion; that was all.

I'll happily remove it anyway though, since it does sort of look too "striking" from the rest.